dynamic-config/config-nacos/src/main/java/org/apache/servicecomb/config/client/ConfigurationAction.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.config.client;
+
+public enum ConfigurationAction {
```

Comment:
You used same package and class name with config-appolo. I Suggest you refactor all code to use a different package name for all class. e.g. 
```
org.apache.servicecomb.config.nacos
```
And we will handle other implentations, maybe in future.

Reply:
The package name of config-nacos model has replaced with "org.apache.servicecomb.config.nacos".